<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Head</class>
 <widget class="QMainWindow" name="Head">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sword Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../applications/cloud_client/main.qrc">
    <normaloff>:/logo32x32.png</normaloff>:/logo32x32.png</iconset>
  </property>
  <widget class="QWidget" name="central">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="edit_controls">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="toggle_items">
         <property name="text">
          <string>Toggle All</string>
         </property>
         <property name="icon">
          <iconset resource="../../applications/cloud_client/main.qrc">
           <normaloff>:/go-down.png</normaloff>:/go-down.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove_items">
         <property name="toolTip">
          <string>Remove Selection</string>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="icon">
          <iconset resource="../../applications/cloud_client/main.qrc">
           <normaloff>:/edit-delete.png</normaloff>:/edit-delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>120</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="filter">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>22</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Filter...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_filter">
         <property name="toolTip">
          <string>Clear Filter</string>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="icon">
          <iconset resource="../../applications/cloud_client/main.qrc">
           <normaloff>:/edit-clear.png</normaloff>:/edit-clear.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeView" name="items">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="allColumnsShowFocus">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="status"/>
 </widget>
 <resources>
  <include location="../../applications/cloud_client/main.qrc"/>
 </resources>
 <connections/>
</ui>
